docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -287,7 +287,7 @@ jobs:
     #   SEMGREP_TIMEOUT: 300
 
     docker:
-      - image: returntocorp/semgrep-agent:v1
+      - image: returntocorp/semgrep-agent:v1 TODO
```

Comment:
Copy of previous comment: Please, what should these configs look like now? Just `returntocorp/semgrep`?

The :v1 is there to demonstrate how to use only a specific version of `semgrep-agent`? Is this recommended? @brendongo @underyx